playground/compiler/package.json

```diff
@@ -8,15 +8,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "react": "^19.0.0-rc-fb9a90fa48-20240614",
-    "react-dom": "^19.0.0-rc-fb9a90fa48-20240614"
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0"
   },
   "devDependencies": {
     "@babel/plugin-transform-react-jsx-development": "^7.25.9",
-    "@types/react": "^18.3.12",
-    "@types/react-dom": "^18.3.1",
+    "@types/react": "^18.3.20",
+    "@types/react-dom": "^18.3.6",
     "@vitejs/plugin-react": "workspace:*",
     "babel-plugin-react-compiler": "0.0.0-experimental-dc8bd44-20241121",
```

Comment:
(Not related to this PR) Should we use beta versions instead?

Reply:
I didn't know there was a beta version. If it's more recent than this one yes we should

Reply:
I'll make a separate PR for that 👍